Structural update for Presentation Flow and Relying Party Solution sections (#601)
italia/eid-wallet-it-docs: docs/en/pid-eaa-presentation.rst

.. include:: ../common/common_definitions.rst

.. _pid-eaa-presentation.rst:

PID/(Q)EAA Presentation

> +++++++++++++++++++

Reviewer: ```suggestion
++++++++++++++++++++++++
```

This section describes how a remote Relying Party Instance requests to a Wallet Instance the presentation of the PID/EAAs.

In this section the following flows are described:

> - :ref:`Remote Flow`, where the User presents a Credential to a remote Relying Party according to `OpenID4VP`_ Draft 20. In this scenario the user-agent and the Wallet Instance can be used in the same device (**Same Device Flow**), or in different devices (**Cross Device Flow**).

Reviewer: ```suggestion
- :ref:`Remote Flow`, where the User presents a Credential to a remote Relying Party according to `OpenID4VP`_. In this scenario the user-agent and the Wallet Instance can be used in the same device (**Same Device Flow**), or in different devices (**Cross Device Flow**).
```

we do not use Draft versions in the main text, we have decided to only define draft versions in their references definition page, see: https://github.com/italia/eid-wallet-it-docs/blob/versione-corrente/docs/common/standards.rst?plain=1#L18

todate, that draft version 23 would became 24.

> - :ref:`Proximity Flow`, where the User presents a Credential to a Verifier App according to ISO 18013-5. The User interacts with a Verifier using proximity connection technologies such as QR Code and Bluetooth Low Energy (BLE).

Reviewer: ```suggestion
- :ref:`Proximity Flow`, where the User presents a Credential to a Verifier App according to `ISO18013-5`_. The User interacts with a Verifier using proximity connection technologies such as using QR Codes and Bluetooth Low Energy (BLE).
```
